test/Ethernet.cpp

```diff
@@ -0,0 +1,10 @@
+#include <Arduino.h>
+#include <ArduinoUnitTests.h>
+
+#include "Ethernet_TC.h"
+
+unittest(test) {
```

Comment:
It seems that we should be able to do _some_ testing! Get a reference to the singleton, query for the IP, renew the lease, etc.